frontend/src/app/Settings/Facility/Components/ManageDevices.test.tsx

```diff
@@ -172,50 +99,26 @@ describe("ManageDevices", () => {
     it("adds a device to the list", async () => {
       const dropdowns = await screen.findAllByRole("combobox");
 
-      expect(dropdowns.length).toBe(4);
+      expect(dropdowns.length).toBe(2);
 
       fireEvent.click(screen.getByText("Add device", { exact: false }));
 
       const updatedDropdowns = await screen.findAllByRole("combobox");
-      expect(updatedDropdowns.length).toBe(6);
+      expect(updatedDropdowns.length).toBe(3);
     });
 
     it("removes a device from the list", async () => {
       const dropdowns = await screen.findAllByRole("combobox");
 
-      expect(dropdowns.length).toBe(4);
+      expect(dropdowns.length).toBe(2);
 
       fireEvent.click(
         screen.getAllByLabelText("Delete device", { exact: false })[0]
       );
 
+      await new Promise((resolve) => setTimeout(resolve, 1000));
       const updatedDropdowns = await screen.findAllByRole("combobox");
-      expect(updatedDropdowns.length).toBe(2);
-    });
-
-    it("only swab types configured for use with a specific device appear in dropdown", async () => {
-      // Row is populated via props; check that only swab types in the dropdown are
-      // allowable for the device
-      const deviceDropdownElement = screen.getByTestId(
-        "device-dropdown-0"
-      ) as HTMLSelectElement;
-
-      expect(
-        (screen.getAllByRole("option", {
-          name: "Fake Specimen 0",
-        })[0] as HTMLOptionElement).selected
-      ).toBeTruthy();
-
-      // Change device to one with _different_ configured swab types
-      userEvent.selectOptions(deviceDropdownElement, "device-b");
-
-      expect(
-        ((
-          await screen.findAllByRole("option", {
-            name: "Fake Specimen 1",
-          })
-        )[0] as HTMLOptionElement).selected
-      ).toBeTruthy();
+      expect(updatedDropdowns.length).toBe(1);
```

Comment:
The amount of red in this test and the component correlates with how much simpler the flow will be for users :100:

Reply:
Oh can't wait till the follow up cleanup story!

Reply:
The end goal is to treat the `DeviceSpecimenType` as the join table it is, and not as a data type. there will be another story coming up to remove the reference to `DeviceSpecimenType` from the `TestEvent` and refer to the device and the specimen type directly.
After that work is done, there will be another story that will make `TestOrder` a temporary object, aka will be deleted once a `TestEvent` is created, this reduce the amount of duplication in the db and cleanup our data.